packages/botonic-react/src/webchat/header.jsx

```diff
@@ -42,16 +54,24 @@ export const DefaultHeader = props => (
       <HeaderTitle>Botonic</HeaderTitle>
       <Subtitle>Online</Subtitle>
```

Comment:
By default, I think that we should not define a subtitle, because 'Online' can be seen as if you are in office hours it's online, if not they might expect to change it.